Packs/Wiz/Integrations/Wiz/Wiz.py

```diff
@@ -571,6 +599,776 @@ def get_resource(resource_id):
+            updateIssue(input: { id: $issueId, patch: $patch, override: $override }) {
+                issue {
+                    id
+                    note
+                    status
+                    dueAt
+                    resolutionReason
+                }
+            }
+        }
+    """)
+
+    try:
+        response = checkAPIerrors(query, variables)
+    except DemistoException:
+        demisto.debug(f"could not find Issue with ID {issue_id}")
+        return {}
+
+    if 'errors' in response:
+        demisto.error(f"Could not find Issue with ID {issue_id}")
+        demisto.error(f"Error: {response}")
+        return (f"Could not find Issue with ID {issue_id}")
+
+    return response
+
+
+def set_issue_note(issue_id, note):
+    """
+    Set a note on Wiz Issue
+    """
+
+    demisto.debug("set_issue_note, enter")
+
+    if not issue_id and not note:
+        demisto.error("You should pass an Issue ID and note.")
+        return "You should pass an Issue ID and note."
+
+    # get existing note to append to it.
+    issue_variables = {
+        'issueId': issue_id
+    }
+    issue_query = (  # pragma: no cover
+        """
+        query IssueDrawer($issueId: ID!) {
+          issue(id: $issueId) {
+            id
+            note
+          }
+        }
+    """)
+
+    try:
+        issue_response = checkAPIerrors(issue_query, issue_variables)
+    except DemistoException:
+        demisto.debug(f"could not find Issue with ID {issue_id}")
+        return {}
+
+    if 'errors' in issue_response:
+        demisto.error(f"Could not find Issue with ID {issue_id}")
+        demisto.error(f"Error: {issue_response}")
+        return (f"Could not find Issue with ID {issue_id}")
+
+    # Check if note is empty for appending
+    issue_response_json = issue_response['data']
+    if not issue_response_json['issue']['note']:
+        new_note = datetime.today().strftime('%Y-%m-%d') + " " + note
+    else:
+        new_note = issue_response_json['issue']['note'] + "\n" + datetime.today().strftime('%Y-%m-%d') + " " + note
+
+    variables = {
+        'issueId': issue_id,
+        'patch': {
+            'note': new_note
+        }
+    }
+    query = (  # pragma: no cover
+        """
+        mutation UpdateIssue(
+            $issueId: ID!
+            $patch: UpdateIssuePatch
+            $override: UpdateIssuePatch
+            ) {
+            updateIssue(input: { id: $issueId, patch: $patch, override: $override }) {
+                issue {
+                    id
+                    note
+                    status
+                    dueAt
+                    resolutionReason
+                }
+            }
+        }
+    """)
+
+    try:
+        response = checkAPIerrors(query, variables)
+    except DemistoException:
+        demisto.debug(f"Could not update Issue with ID {issue_id}")
+        return {}
+
+    return response
+
+
+def clear_issue_note(issue_id):
+    """
+    Clear a note from a Wiz Issue
```

Comment:
Does it clear all the notes?
If so then:
```suggestion
    Clear All notes from a Wiz Issue
```